app/notifications/new_assignment_notification.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class NewAssignmentNotification < Noticed::Base
+  deliver_by :database, association: :noticed_notifications
+
+  def message
+    # user = params[:group_member.user]
+    assignment = params[:assignment]
+    #  t("users.notifications.new_assignment_notification", assignment: assingment.name)
+    t("users.notifications.new_assignment_notification", assignment: assignment)
+  end
+
+  def icon
+    "far fa-star fa-thin"
```

Comment:
```suggestion
    "fa fa-clipboard"
```

Try this one
cc @nitin10s